Presentation.Web/idp-metadata/kombit-saml_demo-brugervendtsystem.xml

```diff
@@ -0,0 +1,39 @@
+<EntityDescriptor ID="_31da2ae2-9db6-4b55-a7e0-a87f3bf7e73c" entityID="https://saml.adgangsstyring.eksterntest-stoettesystemerne.dk" xmlns="urn:oasis:names:tc:SAML:2.0:metadata">
```

Comment:
Det her er metadata for test, så vi skal nok have 2 filer:
- sts-adgangsstyring-test.xml
- sts-adgangsstyring-prod.xml

Reply:
Ja, exactly (tænkte dog at vi lige skal have flowet op at køre før vi gentager med prod)

Reply:
Yes men lad os få de to metadata filer ind :)